### PR TITLE
Fix TestEventsImageImport racy

### LINKED ISSUE #12499: TestEventsImageImport is racy
```
--- FAIL: TestEventsImageImport (0.24s)
        docker_cli_events_test.go:240: Missing import event - got:"2015-04-17T22:16:15.000000000Z 82ad018135c07d590f75ed3315b840e1fc69465e99fb8db2a68c0848e5919662: (from busybox) die"
```

### PR DESCRIPTION
**Pull request by runcom**
Fixes: #12499

Signed-off-by: Antonio Murdaca me@runcom.ninja

### CODE REVIEW

**Comment by runcom** [comment]
ping @LK4D4

File: integration-cli/docker_cli_events_test.go [hunk 1]
```diff
@@ -213,6 +213,44 @@ func TestEventsImageImport(t *testing.T) {
...
+		matchImport := regexp.MustCompile(containerID + `: import$`)
+
+		scanner := bufio.NewScanner(stdout)
+		for scanner.Scan() {
+			if matchImport.MatchString(scanner.Text()) {
+				close(eventImport)
+			}
+		}
+
+		err = eventsCmd.Wait()
```
Comment (LK4D4) [inline integration-cli/docker_cli_events_test.go]: I don't think that here is place for this. You can move `Kill` and wait to end of test.

File: integration-cli/docker_cli_events_test.go [hunk 2]
```diff
@@ -213,6 +213,44 @@ func TestEventsImageImport(t *testing.T) {
...
+		if err != nil {
+			t.Fatalf("failed to start 'docker events': %s", err)
+		}
+
+		go func() {
+			<-finish
+			eventsCmd.Process.Kill()
+		}()
+
+		containerID := <-id
```
Comment (LK4D4) [inline integration-cli/docker_cli_events_test.go]: I think goroutine should start here.

**Comment by LK4D4** [comment]
Note, that Fatal won't work in goroutines properly.

**Comment by runcom** [comment]
yep remember you told me once

File: integration-cli/docker_cli_events_test.go (hunk 1 above)
Comment (runcom) [inline integration-cli/docker_cli_events_test.go]: I don't really undestand this, you mean declare eventsCmd outside of the goroutine?

Comment (LK4D4) [inline integration-cli/docker_cli_events_test.go]: Yes, and start it there too. Only pull stdout in goroutine.

**Comment by LK4D4** [comment]
LGTM

**Comment by jfrazelle** [comment]
LGTM

**Pull request merged**